src/Microsoft.Framework.Runtime/DependencyManagement/LockFileFormat.cs

```diff
@@ -243,7 +243,7 @@ private IPackageFile ReadPackageFile(string property, JToken json)
         private JProperty WritePackageFile(IPackageFile item)
         {
             var json = new JObject();
-            return new JProperty(item.Path, new JObject());
+            return new JProperty(item.Path.Replace('\\', '/'), new JObject());
```

Comment:
We do have several places using this
![untitled](https://cloud.githubusercontent.com/assets/1383883/6423898/5aabe9f0-bea2-11e4-9da0-1e99c8521418.png)

I can add a method to `NuGet.PathUtility`. Does `PathUtility.GetForwardSlashSeparatedPath(string path)` sound good?